container-runtime/ecr-credential-provider/pkg.yaml

```diff
@@ -10,21 +10,24 @@ steps:
         sha256: 0bd75a18faab15dae7a603e1d6b57691953f1b6427f6eaa4c14e9fbfc1a75348
         sha512: e85b0c4a79f6c087147a5868520693f7cb98a471d5497dcda27a6e341e719868f5308ccf8d9781e601ff1b52f332a0447033a096c929e768553e937e154db25e
     env:
-      GOPATH: /go
+      GOPATH: /tmp/go
```

Comment:
i think we can just use `go` so it runs in the builddir and not dependent on internals

Reply:
Might be the case as well, I guess (if you mean relative path).

Reply:
yeh I meant relative, not a big deal

Reply:
#34 0.154 go: GOPATH entry is relative; must be absolute path: "go".

Reply:
/tmp is guaranteed by tools, so shouldn't be that big of an issue